.github/workflows/e2e_windows.yaml

```
@@ -19,6 +19,18 @@ jobs:
   e2e-test-windows:
     runs-on:
       labels: amplify-flutter_windows-latest_8-core
+    strategy:
+      # Allows other matrix items to run if one fails
+      fail-fast: false
+      matrix:
+        channel:
+          - "beta"
+          - "stable"
+        # Skips e2e tests against beta on PRs, except for auth/authenticator
+        # Todo(Jordan-Nelson): Update tests to skip beta for auth once flutter 3.16.0 becomes stable.
+        exclude:
+          - channel: ${{ (github.event_name == 'pull_request' && inputs.package-name != 'amplify_auth_cognito_example' && inputs.package-name != 'amplify_authenticator_example' && 'beta') || 'NONE' }}
+
```

I'm not sure if my suggestion is correct or not but I think the value for `- channel:` should be either 'beta', 'stable' or 'NONE' and so the code to be:

```suggestion
          - channel: ${{ (github.event_name == 'pull_request' && inputs.package-name != 'amplify_auth_cognito_example' && inputs.package-name != 'amplify_authenticator_example') && 'beta' || 'NONE' }}

```

---

In this case it should be the same thing.
- `(true && 'beta') || 'NONE` == 'beta'
- `(true) && 'beta' || 'NONE'` == 'beta'
- `(false && 'beta') || 'NONE` == 'NONE'
- `(false) && 'beta' || 'NONE'` == 'NONE'